### PR TITLE
mimic - qa/tests: Set ansible-version: 2.4

### PR DESCRIPTION
**Pull request by yuriw**
Fixes http://tracker.ceph.com/issues/24926
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
http://pulpito.ceph.com/yuriw-2018-07-18_23:52:41-ceph-ansible-mimic-distro-basic-ovh/

**Comment by Firefishy** [comment]
Also see: https://github.com/ceph/teuthology/pull/1191

File: qa/suites/ceph-ansible/smoke/basic/2-ceph/ceph_ansible.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ meta:
 overrides:
    ceph_ansible:
      vars:
+        ansible-version: 2.4
```
Comment (Firefishy) [inline qa/suites/ceph-ansible/smoke/basic/2-ceph/ceph_ansible.yaml]: This entry isn't being used and the new ansible 2.5 default from https://github.com/ceph/teuthology/pull/1191 is being used instead.

Comment (yuriw) [reply qa/suites/ceph-ansible/smoke/basic/2-ceph/ceph_ansible.yaml]: that's why we are seeing failures, @vasukulkarni ?

Comment (Firefishy) [reply qa/suites/ceph-ansible/smoke/basic/2-ceph/ceph_ansible.yaml]: I believe the issue is the ```ansible-version:``` needs to be moved up one level equal to ```vars:``` per this line: https://github.com/ceph/teuthology/blob/af590b821a72d8a60de794f1ae6fe700ae7a055b/teuthology/task/ceph_ansible.py#L28

Comment (yuriw) [reply qa/suites/ceph-ansible/smoke/basic/2-ceph/ceph_ansible.yaml]: @Firefishy you mean like this?

 overrides:
    ceph_ansible:
      ansible-version: 2.4
      vars:

Comment (Firefishy) [reply qa/suites/ceph-ansible/smoke/basic/2-ceph/ceph_ansible.yaml]: Yes that is what I meant.

**Comment by yuriw** [comment]
testing http://pulpito.front.sepia.ceph.com:80/yuriw-2018-07-20_22:47:40-ceph-ansible-mimic-distro-basic-ovh/

**Comment by Firefishy** [comment]
Testing url: http://pulpito.ceph.com/yuriw-2018-07-20_22:47:40-ceph-ansible-mimic-distro-basic-ovh/

File: qa/suites/ceph-ansible/smoke/basic/2-ceph/ceph_ansible.yaml (hunk 1 above)
Comment (yuriw) [reply qa/suites/ceph-ansible/smoke/basic/2-ceph/ceph_ansible.yaml]: @Firefishy looks like this is not correct

**Comment by yuriw** [comment]
@Firefishy used as suggested 

> (05:09:44 PM) Firefishy: yuriw: ah, yaml, sometimes depending on the parser just enough to quote the number. "2.4"

Seems ok now :+1: 

http://pulpito.ceph.com/yuriw-2018-07-21_00:49:05-ceph-ansible-mimic-distro-basic-ovh/

**Comment by yuriw** [comment]
@theanalyst @smithfarm  FYI

**Pull request merged**